packages/web-components/src/components/ic-chip/ic-chip.tsx

```diff
@@ -83,6 +90,10 @@ export class Chip {
   componentWillLoad(): void {
     removeDisabledFalse(this.disabled, this.el);
 
+    if (!this.dismissible && this.customColor !== null) {
+      this.setChipColour();
```

Comment:
this will only happen when the component first loads. should there be a @watch on the customColour prop to run the same logic if the value changes?

Reply:
if so, should also add a test to check it works when the prop is changed